### PR TITLE
Issue 44 create login modal

### PR DESCRIPTION
**Pull request by mamohamed252**
# PR Description
Please include a summary of the change.
Created dialog components for login and created login form without much styling. Also created a test to see if the button exists.

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [X] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] This change requires a documentation update

## Aditonal Context or Relevant Issues
Please include additional context.

## Issue Link
Please link the original issue URL.
https://github.com/InfAurora/TypeScript-Projects/issues/44

# Checklist:
- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: simple-blog/src/app/app.component.spec.ts [hunk 1]
```diff
@@ -1,8 +1,10 @@
 import { TestBed } from '@angular/core/testing';
-import { RouterTestingModule } from '@angular/router/testing';
 import { AppComponent } from './app.component';
-import { MatToolbarModule } from '@angular/material/toolbar';
 import { BlogNavbarComponent } from './blog-navbar/blog-navbar.component';
+import { MatDialogModule } from '@angular/material/dialog'
+import { MatToolbarModule } from '@angular/material/toolbar';
+import { RouterTestingModule } from '@angular/router/testing';
```
Comment (mamohamed252) [inline simple-blog/src/app/app.component.spec.ts:6]: Fix ordering by alphabetizing as well!

File: simple-blog/src/app/app.component.spec.ts [hunk 2]
```diff
@@ -15,8 +17,9 @@ describe('AppComponent', () => {
         BlogNavbarComponent,
       ],
       imports: [
-        RouterTestingModule,
+        MatDialogModule,
         MatToolbarModule,
+        RouterTestingModule
```
Comment (mamohamed252) [inline simple-blog/src/app/app.component.spec.ts:22]: alphabetizing

File: simple-blog/src/app/app.module.ts [hunk 3]
```diff
@@ -1,30 +1,35 @@
...
 import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { BrowserModule } from '@angular/platform-browser';
 
 import { MatButtonModule } from '@angular/material/button';
-import { MatToolbarModule } from '@angular/material/toolbar';
-import { MatInputModule } from '@angular/material/input';
+import { MatDialogModule } from '@angular/material/dialog'
 import { MatIconModule } from '@angular/material/icon';
+import { MatInputModule } from '@angular/material/input';
+import { MatToolbarModule } from '@angular/material/toolbar';
```
Comment (mamohamed252) [inline simple-blog/src/app/app.module.ts:10]: alphabetizing

File: simple-blog/src/app/blog-navbar/blog-navbar.component.html [hunk 4]
```diff
@@ -15,7 +15,7 @@
     </button>
   </div>
   <div id="nav-content">
-    <button type="button" aria-label="Login">Login</button>
```
Comment (InfAurora) [inline simple-blog/src/app/blog-navbar/blog-navbar.component.html:18]: We want to keep the aria label for accessibility

**Review by InfAurora — APPROVED** [review]

**Pull request merged**